### PR TITLE
Change 9f2 from rounding to truncation

### PR DESCRIPTION
**Pull request by ohexter**
Change 9f2 from rounding to truncation

### CODE REVIEW

File: wca-guidelines.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # <wca-title>WCA Guidelines
 
-<version>Version: January 1, 2024
+<version>Version: January 1, 2025
```
Comment (Nevseros) [inline wca-guidelines.md]:
```suggestion
<version>Version: January 1, 2024
```

File: wca-regulations.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # <wca-title>WCA Regulations
 
-<version>Version: January 1, 2024
+<version>Version: January 1, 2025
```
Comment (Nevseros) [inline wca-regulations.md]:
```suggestion
<version>Version: January 1, 2024
```

**Comment by CarterKoala** [comment]
We changed this before from "Truncated to the nearest" to it's current state, since "to the nearest" only really applies to rounding. Any particular reason for that change?

File: wca-regulations.md [hunk 3]
```diff
@@ -246,8 +246,8 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 9b5a) Full round formats for this event: "Best of X" (where X is 1, 2, or 3).
         - 9b5b) Cutoff formats for this event: "Best of X" (where X is 1 or 2) cutoff phase for "Best of Y" (where Y is 2 or 3, and Y > X).
 - 9f) The results of a round are measured as follows:
-    - 9f1) All timed results under 10 minutes, except for 3x3x3 Multi-Blind, are measured and truncated down to hundredths of a second. All timed averages and means under 10 minutes are measured and rounded to the nearest hundredth of a second.
-    - 9f2) All timed results, averages, and means over 10 minutes, as well as all times for 3x3x3 Multi-Blind results, are measured and rounded to the nearest second (e.g. X.49 becomes X, X.50 becomes X+1).
+    - 9f1) All timed results under 10 minutes, except for 3x3x3 Multi-Blind, are measured and truncated to the nearest hundredth of a second. All timed averages and means under 10 minutes are measured and rounded to the nearest hundredth of a second.
```
Comment (ohexter) [inline wca-regulations.md]:
```suggestion
    - 9f1) All timed results under 10 minutes, except for 3x3x3 Multi-Blind, are measured and truncated to hundredths of a second. All timed averages and means under 10 minutes are measured and rounded to the nearest hundredth of a second.
```

File: wca-regulations.md [hunk 4]
```diff
@@ -246,8 +246,8 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
+    - 9f2) All timed results, averages, and means over 10 minutes, as well as all times for 3x3x3 Multi-Blind results, are measured and truncated to the nearest second.
```
Comment (ohexter) [inline wca-regulations.md]:
```suggestion
    - 9f2) All timed results, averages, and means over 10 minutes, as well as all times for 3x3x3 Multi-Blind results, are measured and truncated to seconds.
```

**Review by ohexter — COMMENTED** [review]
Fine by me, as long as we lose "truncated down". Potential to use "full seconds" in 9f2 if this adds further clarity.

**Pull request merged**